tests/benchmarks/micro_benchmarks/test_ib_traffic_performance.py

```diff
@@ -245,14 +262,15 @@ def test_ib_traffic_performance(self, mock_gpu):
         # Positive case - valid raw output.
         os.environ['OMPI_COMM_WORLD_RANK'] = '0'
         assert (benchmark._process_raw_result(0, raw_output_0))
+        assert (benchmark._process_raw_result(0, raw_output_1))
```

Comment:
can you check `benchmark.result[metric]` also gives a list of -1 instead of -0.125? the test case should only work with your fix (before the fix it would fail)